_posts/2019-05-02-zarr-2.3-release.md

```diff
@@ -0,0 +1,134 @@
+---
+layout: post
+title:  "Zarr Python 2.3 release"
+date:   2019-05-02
+categories: zarr python release
+---
+
+Recently we released version 2.3 of the Python Zarr package, which
+implements the Zarr protocol for storing N-dimensional typed arrays,
+and is designed for use in distributed and parallel computing. This
+post provides an overview of new features in this release, and some
+information about future directions for Zarr.
+
+## New storage options for distributed computing
+
+A key feature of the Zarr protocol is that the underlying storage
+system is decoupled from other components via a simple key/value
+interface. In Python, this interface corresponds to the
+[`MutableMapping` interface](@@TODO), which is the interface that
+Python dictionaries (`dict`) implement. The simplicity of this
+interface means it is relatively straightforward to add support for a
+range of different storage systems. The 2.3 release adds support for
+storage using [SQLite](@@TODO), [Redis](@@TODO), [MongoDB](@@TODO) and
+[Azure Blob Storage](@@TODO).
```

Comment:
```suggestion
[Azure Blob Storage]( https://zarr.readthedocs.io/en/stable/api/storage.html#zarr.storage.ABSStore ).
```

Note: Sorry for the second comment like this. GitHub didn't allow me to consolidate them.